src/frontend/fir_visitor.cpp

```diff
@@ -331,6 +331,10 @@ namespace graphit {
             for (auto genericArg : expr->genericArgs) {
                 genericArg->accept(this);
             }
+
```

Comment:
The visitors also need to be modified to have calls to accept for rest of the args (from, to ...)